test_fms/mpp/test_mpp_domains2.sh

```diff
@@ -115,19 +127,19 @@ then
     echo "Does not work on Darwin"
 else
     sed "s/test_interface = .false./test_interface = .true./" $top_srcdir/test_fms/mpp/input_base.nml > input.nml
-    mpirun -n 2 ./test_mpp_domains
+    $mpi_launcher $npes ./test_mpp_domains
 fi
 
 #echo "14: Test Check Parallel"
 #echo "Does not work on Darwin or elsewhere"
 #sed "s/check_parallel = .false./check_parallel = .true./" $top_srcdir/test_fms/mpp/input_base.nml > input.nml
-#mpirun -n 6 ./test_mpp_domains
+#$mpi_launcher -n 6 ./test_mpp_domains
 
 echo "15: Test Get Nbr"
 if [ "$skip_test" = "true" ]
 then
     echo "Does not work on Darwin"
 else
     sed "s/test_get_nbr = .false./test_get_nbr = .true./" $top_srcdir/test_fms/mpp/input_base.nml > input.nml
-    mpirun -n 2 ./test_mpp_domains
+    $mpi_launcher $npes ./test_mpp_domains
```

Comment:
Since this is something that is done frequently, perhaps writing a shell function to generate the run command would be a better idea.  It would make it easier for new tests.

Reply:
@underwoo Do you still want @uramirez8707 to write a separate shell function, or should I dismiss this request and merge?